libraries/default.rb

```diff
@@ -40,8 +40,43 @@ def self.error_message(error_code)
     end
   end
 
+  class ZookeeperDynamicConfig < Hash
+    # FIXME: client port is hardcoded in this cookbook
+    # so it's unlikely working if Zk client port is different or
+    # if there are several Zk instances on same node. We do so
+    # because our recipes don't send client port info for now.
+    CLIENT_PORT = 2181
+
+    # Fields we ignore when comparing
+    IMMUTABLE_FIELDS = %w(version).freeze
```

Comment:
OPT: why not calling that constant "IGNORED_FIELDS" then?

Reply:
done